css/parser/src/lexer/value.rs

```diff
@@ -67,56 +67,57 @@ where
 
         self.input.bump(); // ' or "
 
-        let mut was_escape = false;
         loop {
-            if !was_escape {
-                let chunk = self.input.uncons_while(|peeked| match peeked {
-                    '\'' | '"' => peeked != quote,
-                    '\\' => false,
-                    _ => true,
-                });
-
-                buf.push_str(chunk);
-            }
```

Comment:
Faster than before

Reply:
Did you run the benchmark?
Seems like the new code is appending one char at a time, so I'm not sure how can it be faster than before.

Reply:
@kdy1 we don't look up above, but anyway for valid parsing escapes we should use this logic, maybe we can optimize it better in future

Reply:
I see, then it's fine.